src/contracts/types/ConfigTypes.sol

```diff
@@ -87,7 +87,7 @@ enum CallConfigIndex {
     DelegateUser,
     PreSolver,
     PostSolver,
-    RequirePostOpsCall,
+    RequirepostOpsDelegateCall,
```

Comment:
Should probably be `requirePostOps`

struct `CallConfig` also uses that, see line 41
```solidity
struct CallConfig {
    ...
    bool requirePostOps;
    ...    
}
```

Reply:
ok, no problem   (it would have probably required updating other documentation as well)